.changelog/1740.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+builtin/aws: add ability to pass additional security groups for ecs
```

Comment:
```suggestion
```release-note:improvement
plugin/aws: add ability to pass additional security groups for ecs
```